src/utility.js

```diff
@@ -0,0 +1,61 @@
+export const formatDateTime = (timestamp) => {
+  const date = new Date(timestamp);
+
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const formattedTime = `${hours}:${minutes < 10 ? "0" : ""}${minutes}`;
+
+  const day = date.getDate();
+  const month = date.getMonth() + 1; // Month is zero-based, so we add 1
+  const year = date.getFullYear();
+  const formattedDate = `${day}/${month}/${year}`;
+
+  const formattedDateTime = `${formattedTime} ${formattedDate}`;
+  return formattedDateTime; // Output: 16:20 25/4/2024
+};
+
+export function generateRandomNumber() {
+  return Math.floor(Math.random() * 5); // Generates a random integer between 0 and 4
+}
+
+export function getStatusColor(status) {
+  switch (status) {
+    case "Delayed":
+      return "#FF5733"; // Red or orange color for delayed status
```

Comment:
this should instead return className, and those classNames should have css in the css files.